src/geometry_base/CMakeLists.txt

```diff
@@ -3,9 +3,11 @@
 ##############################################################################
 set(SOURCES
     branch_point.cc
-    brute_force_find_road_positions_strategy.cc
+    brute_force_find_road_positions_strategy
```

Comment:
This file is being used by brute_force_find_road_positions_test.cc, should we leave this modification (since it is not just delete and done) to a follow up PR regarding the test of BruteForceStrategy/KDTreeStrategy? Or I could delete the proposed file and the test file aswell. Let me know what you think.